packages/common/src/HardhatConfig.js

```diff
@@ -29,6 +29,9 @@ function getHardhatConfig(configOverrides, workingDir = "./", includeTruffle = t
     settings: { optimizer: { enabled: true, runs: 200 } },
   };
 
+  // Some tests in the /test directory should not be tested using hardhat. Define these tests to ignore.
+  const testBlacklist = ["insured-bridge/e2e"];
```

Comment:
@daywiss employed a strategy that I thought was really effective for designing "integration" tests in-line with other tests -- by using an extension: `*.e2e.js`. These get ignored in the normal test command.

Nothing to change now, but it might be ideal to have an easy rule to distinguish integration tests from normal hardhat tests (maybe e2e file path or extension) and have a separate pre-built command to run just those integration tests (if there are different sub-types that need diff environments, then a way to run each type).

Reply:
that's a _great_ idea. I was trying to think of a simple way to consistently ignore files that have a particular naming style. putting *.e2e.js` makes a lot of sense! I'll update this accordingly.